sp800-63a/sec5_proofing.md

```diff
@@ -6,18 +6,18 @@
 
 _This section is normative._
 
-This section lists the requirements to resolve, validate, and verify an identity and any supplied identity evidence. The requirements are intended to ensure the claimed identity is the actual identity of the subject attempting to enroll with the CSP and that scalable attacks affecting a large population of enrolled individuals require a greater time and cost than the value of the resources the system is protecting.
+This section lists the requirements to resolve, validate, and verify an identity and any supplied identity evidence. The requirements are intended to ensure the claimed identity is the actual identity of the subject attempting to enroll with the CSP and that scalable attacks affecting a large population of enrolled individuals require greater time and cost than the value of the resources the system is protecting.
 
 ### <a name="resolve"></a>5.1 Identity Resolution
 
-The goal of identity resolution is to uniquely distinguish an individual within a given population or context.  Effective identity resolution uses the smallest set of attributes necessary to resolve to a unique individual. It provides the CSP an important starting point in the overall identity proofing process, to include the initial detection of potential fraud, but in no way represents a complete and successful identity proofing transaction.  
+Identity resolution's goal is to uniquely distinguish an individual within a given population or context. Effective identity resolution uses the smallest set of attributes necessary to resolve to a unique individual. It provides the CSP an important starting point in the overall identity proofing process, to include the initial detection of potential fraud, but in no way represents a complete and successful identity proofing transaction.
```

Comment:
This one bugs me a little bit.  Identity resolution isn't a noun. I'll defer but it seems odd to me.